tests/integration/devfile/cmd_devfile_deploy_test.go

```diff
@@ -110,4 +111,27 @@ var _ = Describe("odo devfile deploy command tests", func() {
 			})
 		})
 	})
+
+	When("recording telemetry data", func() {
+		BeforeEach(func() {
+			helper.CreateTelemetryDebugFile()
+			helper.Cmd("odo", "deploy").AddEnv("PODMAN_CMD=echo").ShouldPass()
+			// An ENV file should have been created indicating current namespace
+			Expect(helper.VerifyFileExists(".odo/env/env.yaml")).To(BeTrue())
+			helper.FileShouldContainSubstring(".odo/env/env.yaml", "Project: "+commonVar.Project)
```

Comment:
We probably copied these expects from another test. I don't think we nned them again here, we can keep the test as simple as possible

Reply:
removing